linera-service/src/cli_wrappers/docker.rs

```diff
@@ -65,7 +67,9 @@ impl DockerImage {
             command.args(["--build-arg", &target_arg]);
         }
 
+        let git_head_commit = get_git_head_commit().await?;
```

Comment:
@jvff makes the excellent point that if we have a `binaries` dir, we should call the `linera` binary in there to get this version information — since it might be different to the version information of the current git repository!